src/main/java/frc/robot/shooter/ShootCommand.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.shooter;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class ShootCommand extends Command {
```

Comment:
I think we are going to standardize on not writing `Command` in every command class name. Change to `Shoot`

Reply:
renamed